src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,508 @@
+
+**Acquisition:** In case of studies with multiple acquisitions per subject using different tracers,
+the `acq-<label>` must be used to distinguish between different tracers.
+Please keep in mind that the label used is arbitrary and each file requires
+a separate JSON sidecar with details of the tracer used (see below).
+Examples are `acq-18FFDG` for fludeoxyglucose, `acq-11CPIB` for Pittsburgh compound B, etc.
+
+**Reconstruction:** The reconstruction key (`rec-<label>`) has four reserved values:
+- `acdyn`, for reconstructions with attenuation correction of dynamic data;
+- `acstat`, for reconstructions with attenuation correction of static data;
+- `nacdyn`, for reconstructions without attenuation correction of dynamic data;
+- `nacstat`, for reconstructions without attenuation correction of static data.
+Further details regarding reconstruction are in the `_pet.json` file.
+In case of multiple reconstructions of the data with the same type,
+we allow for using a number after the `<label>` in order to distinguish,
+for example `rec-acdyn1` and `rec-acdyn2`.
+
+The run entity is used if one scan type/contrast is repeated multiple times
+within the same scan session/visit.
+If several scans of the same modality are acquired they MUST be indexed
+with a key-value pair: `_run-1`, `_run-2`, `_run-3` etc. (only integers are allowed as run labels).
+When there is only one scan of a given type the run key MAY be omitted.
+An example of this would be two consecutive scans performed within the same session,
+for example two short FDG scans right after each other.
+It is our assumption that the `run-<label>` is used in PET for the cases
+where the subject does not leave the scanner.
+Otherwise, we refer to the `ses-<label>` definition.
+
+In addition to the imaging data a `_pet.json` sidecar file needs to be provided.
+The included metadata are divided into sections described below.
+
+### PET sidecar JSON (`*_pet.json`)
+
+#### Information
+
+This section is mandatory and contains general information about the imaging experiment.
+Some of the fields are marked OPTIONAL, for example anaesthesia; for those fields
+the BIDS validator will not throw an error even if they are not present.
+Note, although bodyweight is a recommended information in (Knudsen et al. 2020, JCBFM [1]),
+this consists of meta information at the participant level and should hence
+be part of the `participants.tsv` or `session.tsv` file in case of multiple measurements.
+
+| Field name                | Definition                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Adding this as a reminder for later (as I think this might require some synchronization with the validator for the data type) but I suspect that tables should now have the following headings / format. 

Will let my fellow maintainers correct me if I am wrong.

| **Key name**    | **Requirement level** | **Data type** | **Description**     |
| -   | - | - | -    |
| example    | RECOMMENDED           | [string][]    |  some text |

Reply:
yes, thanks @Remi-Gau maybe we can even get this done tomorrow afternoon before we meet.

Reply:
This has been corrected.